src/app/blog/articles.service.ts

```diff
@@ -5,22 +5,22 @@ import { AngularFirestore } from '@angular/fire/firestore';
 import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
 
-import { Blog } from '../shared/models/blog.model';
+import { Article } from '../shared/models/article.model';
 import { DataService } from '../shared/models/data-service.model';
 import { FirebaseCollections } from '../shared/models/collections.constant';
 import { CloudFunctions } from '../shared/models/cloud-functions.constant';
 
 const DEFAULT_IMAGE = 'assets/images/GP4Tech-logo.png';
 
 @Injectable()
-export class BlogsService extends DataService<Blog> {
+export class ArticlesService extends DataService<Article> {
   defaultBlogImage = DEFAULT_IMAGE;
 
   constructor(http: HttpClient, db: AngularFirestore) {
-    super(http, db, FirebaseCollections.blogs);
+    super(http, db, FirebaseCollections.articles);
   }
 
-  getAllBlogs(): Observable<Blog[]> {
+  getAllBlogs(): Observable<Article[]> {
```

Comment:
If we are renaming the blogs object/ services and stff i think we need to rename the module, folders and methods too, not sure what do you think?

Reply:
I think the folder is ok with Blog because it's still a blog, but the blog has artibles, not other blogs, the module should also be the blog module. About the methods, these have to be renamed, my bad, also the collection in the database, but this last one will be updated in another PR. I will change the name of the methods.

Reply:
Also, I will change the name of the cloud functions in another PR

Reply:
Sounds good, let me know when methods are changed to merged this one

Reply:
@dmeneses I renamed methods and variables, please take a look